build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -1341,3 +1341,4 @@ CONFIG_IOTBUS_ADC=y
 # Security Framework
 #
 # CONFIG_SECURITY_API is not set
+CONFIG_PROD_HEADER="prod/fac.h"
```

Comment:
And by menuconfig, here is a wrong location. Please add this under `Board-Specific Options` tab.

Reply:
it's temporary fix. so I'll fix it after sync is done